traffic_ops/testing/api/v4/cdn_locks_test.go

```diff
@@ -528,6 +643,52 @@ func TestCDNLocks(t *testing.T) {
 								}
 							})
 						}
+					case "PROFILE POST":
+						{
+							t.Run(name, func(t *testing.T) {
+								alerts, reqInf, err := testCase.ClientSession.CreateProfile(profile, testCase.RequestOpts)
+								for _, check := range testCase.Expectations {
+									check(t, reqInf, nil, alerts, err)
+								}
+							})
+						}
```

Comment:
Yeah the cdn_lock tests got way bigger than intended. I think instead of adding a different test structure I will be moving each of the different endpoint tests that have a cdn lock test back into their own test file. I'll just need to create a prerequisite cdn thats only meant for locks so there is no overlap. I will need to do this as a separate PR though.

Reply:
Okay, but is that a reason not to use this approach?